install/installer/pkg/components/proxy/deployment.go

```diff
@@ -258,6 +264,12 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 								}, {
 									Name:  "WORKSPACE_HANDLER_FILE",
 									Value: strings.ToLower(string(ctx.Config.Kind)),
+								}, {
+									Name:  "ANALYTICS_PLUGIN_TRUSTED_SEGMENT_KEY",
+									Value: trustedSegmentKey,
+								}, {
+									Name:  "ANALYTICS_PLUGIN_UNTRUSTED_SEGMENT_KEY",
+									Value: untrustedSegmentKey,
 								}},
```

Comment:
Can we pull this into something like `common.AnalyticsEnv` similar to other re-usable parts?

Reply:
Maybe I misunderstand, but I think this PR already adds such config: https://github.com/gitpod-io/gitpod/pull/17162/files#diff-590a17333c6831c6ff532edbab0b818ee3c49fb033d163722552bbbfafd31a8cR466

Reply:
You're right, sorry. Missed that.